README.md

```diff
@@ -67,7 +67,7 @@ Please see [SECURITY.md](./SECURITY.md).
 
 | CometBFT version | Requirement | Version        | Tested with |
 |------------------|-------------|----------------|-------------|
-| main             | Go version  | 1.22 or higher | 1.22        |
+| main             | Go version  | 1.22 or higher | 1.22.5      |
```

Comment:
```suggestion
| main             | Go version  | 1.22 or higher | 1.22      |
```

That might create the illusion that we only tested this with 1.22.5, but we also tested CometBFT with other minor Go releases.

Reply:
I understood the column `Tested with` as "we tested with Go versions up to 1.22.5". Maybe we could update it to say "up to 1.22.5" to make it clearer?